packages/mux-player-react/package.json

```diff
@@ -68,19 +81,17 @@
   "license": "MIT",
   "scripts": {
     "clean": "shx rm -rf dist/",
-    "dev:cjs": "npm run build:cjs -- --watch=forever",
```

Comment:
question(blocking): why were all of the dev scripts removed here? this feels out of scope.

Reply:
I sometimes use them. It just makes reviews harder when there are many changes that are not related to the scope of the primary effort, especially when there are lots of changes already. It also makes it harder to assess whether there are implications of these changes that I need to reason through.